requirements.txt

```diff
@@ -1,8 +1,12 @@
+dj-database-url==0.5.0
 Django==2.1.7
 django-cors-middleware==1.3.1
 django-extensions==2.1.6
+django-heroku==0.3.1
 djangorestframework==3.9.1
-psycopg2-binary==2.7.7
```

Comment:
@winniekariuki we might need to keep the `psycopg2-binary` on the requirements file since it replaces the `psycopg2` package.